DonjinKrawler-Server/src/main/resources/application.properties

```diff
@@ -1,2 +1,2 @@
 logging.level=DEBUG
-logging.fileName=yyyyMMdd_krawler_log.log
```

Comment:
Čia kodėl užkomentavai? Jeigu nereikia tai gal ištrink tiesiog.

Reply:
You are goddamn right. Na šiaip gali pasilikti tam, nes nesugalvosiu daugiau tokio gražaus logname formato ;)